src/confscreen.cpp

```diff
@@ -342,6 +347,9 @@ void TextWindow::ShowConfiguration() {
         SS.automaticLineConstraints ? CHECK_TRUE : CHECK_FALSE);
     Printf(false, "  %Fd%f%Ll%s  use turntable mouse navigation%E", &ScreenChangeTurntableNav,
         SS.turntableNav ? CHECK_TRUE : CHECK_FALSE);
+    Printf(false, "  %Fd%f%Ll%s  immediately edit dimensions%E",
```

Comment:
What about `edit newly added dimensions`? The variable naming is fine, but the preference is a bit confusing.